brats/transformations.py

```diff
@@ -251,3 +234,55 @@ def _(mask: torch.Tensor, classes: typing.List[int]) -> torch.Tensor:
     for class_id, label in enumerate(filter(lambda label: label != 0, classes)):
         transformed[class_id][mask[0, ...] == label] = 1
     return transformed
+
+
+class CommonTransformation(abc.ABC):
+    """
+    Interface for transformations that are supposed to transform multiple inputs in the same way.
+    """
+
+    @abc.abstractmethod
+    def __call__(self, imgs: typing.List[typing.Union[np.ndarray, torch.Tensor]]):
+        raise NotImplementedError
+
+
+class RandomCrop(CommonTransformation):
+    def __init__(self, size: typing.Tuple[int, int]):
```

Comment:
The user does not know whether passed tuple should be `(x, y)` or `(y, x)`. Docstring maybe?

Reply:
Docstring added.